changelogs/unreleased/gh-7166-allow-spurious-wakeups-in-cbus_call.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* Allowed spurious wakeup of a fiber, which is waiting for `cbus_call` completion
+  (gh-7166).
```

Comment:
AFAIU this commit introduces no user-visible changes, nor does it fix any bugs. Hence it doesn't need a changelog entry.

Reply:
I've removed this file so far, will add a common changelog entry when I finish fiber_set_cancellable refactoring.